pynestml/utils/logger.py

```diff
@@ -100,8 +100,8 @@ def log_message(cls, neuron=None, code=None, message=None, error_position=None,
         from pynestml.meta_model.ast_source_location import ASTSourceLocation
         assert (neuron is None or isinstance(neuron, ASTNeuron)), \
             '(PyNestML.Logger) Wrong type of neuron provided (%s)!' % type(neuron)
-        assert (error_position is None or isinstance(error_position, ASTSourceLocation)), \
-            '(PyNestML.Logger) Wrong type of error position provided (%s)!' % type(error_position)
+        # assert (error_position is None or isinstance(error_position, ASTSourceLocation)), \
```

Comment:
Why is this commented out?

Reply:
Ah, this is associated with #511, I forgot to remove it, I'll do that

Reply:
Should it be removed our uncommented?

Reply:
Unless you fixed it, I will probably have to remove it, but I'll check it again on both computers... not before the end of the week at best though...